edkrepo/commands/list_pins_command.py

```diff
@@ -121,6 +123,7 @@ def run_command(self, args, config):
                             output_string = separator.join((output_string, 'Parsing Errors: {}'.format(parse_output.strip())))
                         if args.description:
                             output_string = separator.join((output_string, 'Description: {}\n'.format(pin.project_info.description)))
+        sys.stdout = stdout

```

Comment:
This needs to be put in the finally block of a try/finally to prevent an exception from causing this statement to not execute.